### ISSUE TITLE
Update 1.4.10 with vertical scrolling for web content designed to be scrolled horizontally

### ISSUE THREAD

**Issue by MewenLeHo** [Orange-OpenSource/a11y-guidelines#657]
Following the update in the Va11ydette:

> We are missing an important part in our criteria. Indeed WCAG 1.4.10:
> 
> > Content can be presented without loss of information or functionality, and without requiring scrolling in two dimensions for:
> > 
> > Vertical scrolling content at a width equivalent to 320 CSS pixels;
> > **Horizontal scrolling content at a height equivalent to 256 CSS pixels**.
> 
> By doing that we are missing the whole point web about content designed to be scrolled horizontally.
> 
> We also need to add pre-defined issue for this part of the criteria.
